cicd-scripts/app_start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Activating virtual environment..."
+source /home/ec2-user/app/venv/bin/activate
+
+echo "Running searchgov-spider application..."
+# python /home/ec2-user/app/app.py
+
+pkill scrapy
+
+
+nohup scrapy crawl domain_spider &
+nohup scrapy crawl domain_spider_js &
+
```

Comment:
This should be starting scrapyd, scrapydweb, and logparser instead of scrapy directly right?